services/concepts/archive-data.md

```diff
@@ -56,25 +56,26 @@ Infura applies [rate limits](../how-to/avoid-rate-limiting.md) to ensure network
 
 Requests for Ethereum archive data are more expensive and are therefore subject to different rate limits.
 
-**Free plan**
+| Plan               | Archive Requests per Day | Total Requests per Day | Rate Limits                                                            |
+|--------------------|--------------------------|------------------------|------------------------------------------------------------------------|
+| **Free plan**      | 25,000                   | 100,000                | 75,000 standard requests available after 25,000 archive requests.      |
+| **Developer plan** | 100,000                  | 200,000                | 100,000 standard requests available after 100,000 archive requests.    |
 
```

Comment:
```suggestion
```

I don't think this table is necessary because the amount of content is fairly small, and is repeated in its entirety in the following sections.